README.md

```diff
@@ -2840,44 +2840,44 @@ type Foo = { a: Foo, b: Bar }
 <a name="eslint-plugin-flowtype-rules-quotes"></a>
 ### <code>quotes</code>
 
-Enforces single quotes or double quotes.
+Enforces single quotes or double quotes around string literals.
 
 <a name="eslint-plugin-flowtype-rules-quotes-options-3"></a>
 #### Options
 
 The rule has string options of:
 
-* `"double"` (default) requires double quotes around strings.
-* `"single"` requires single quotes around strings.
+* `"double"` (default) requires double quotes around string literals.
```

Comment:
Hm I wonder why is default double? Guess same as regular quotes rule?

Reply:
Yep keeping it standard